### PR TITLE
Upgraded PG to version 10

### PR DESCRIPTION
**Pull request by madarve**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FROM jenkins/jnlp-slave:3.16-1-alpine
 MAINTAINER Infrastructure team <team-infrastructure@newstore.com>
 
-ENV PG_MAJOR 9.6
-ENV PG_VERSION 9.6.5
+ENV PG_MAJOR 10
+ENV PG_VERSION 10.0
```
Comment (ecasti-newstore) [inline Dockerfile]: must be 10.6

**Review by ecasti-newstore — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ecasti-newstore) [reply Dockerfile]: already tested with 10.6 and it work 
_/home/jenkins # pg_dump -V
pg_dump (PostgreSQL) 10.6
/home/jenkins #_

Comment (madarve) [reply Dockerfile]: Fixed in 4f2d03875a55896ce8eab3e322cde949b756168b

**Review by kayabendroth — COMMENTED** [review]
Squash then GTM.

**Pull request merged**